indico/htdocs/js/indico/jquery/categorynavigator.js

```diff
@@ -264,6 +302,20 @@
             }
         },
 
+        refurbishUI: function() {
```

Comment:
Why not refurbish? Refresh would be a wrong name since I am not refreshing anything, just moving things around.